src/plugins/intel_gna/src/transformations/utils/transformation_helper.cpp

```diff
@@ -121,36 +124,74 @@ std::shared_ptr<ngraph::Node> VerifyBiasGetConst(std::shared_ptr<ngraph::Node> c
         auto new_fq = fq_layer->clone_with_new_inputs(
             {last_node,
-             ngraph::opset7::Constant::create(
-                 ngraph::element::f32,
-                 ngraph::Shape{1},
-                 std::dynamic_pointer_cast<ngraph::opset7::Constant>(fq_layer->input_value(1).get_node_shared_ptr())
+             Constant::create(
+                 ov::element::f32,
+                 ov::Shape{1},
+                 std::dynamic_pointer_cast<Constant>(fq_layer->input_value(1).get_node_shared_ptr())
                      ->cast_vector<float>()),
-             ngraph::opset7::Constant::create(
-                 ngraph::element::f32,
-                 ngraph::Shape{1},
-                 std::dynamic_pointer_cast<ngraph::opset7::Constant>(fq_layer->input_value(2).get_node_shared_ptr())
+             Constant::create(
+                 ov::element::f32,
+                 ov::Shape{1},
+                 std::dynamic_pointer_cast<Constant>(fq_layer->input_value(2).get_node_shared_ptr())
                      ->cast_vector<float>()),
-             ngraph::opset7::Constant::create(
-                 ngraph::element::f32,
-                 ngraph::Shape{1},
-                 std::dynamic_pointer_cast<ngraph::opset7::Constant>(fq_layer->input_value(3).get_node_shared_ptr())
+             Constant::create(
+                 ov::element::f32,
+                 ov::Shape{1},
+                 std::dynamic_pointer_cast<Constant>(fq_layer->input_value(3).get_node_shared_ptr())
                      ->cast_vector<float>()),
-             ngraph::opset7::Constant::create(
-                 ngraph::element::f32,
-                 ngraph::Shape{1},
-                 std::dynamic_pointer_cast<ngraph::opset7::Constant>(fq_layer->input_value(4).get_node_shared_ptr())
+             Constant::create(
+                 ov::element::f32,
+                 ov::Shape{1},
+                 std::dynamic_pointer_cast<Constant>(fq_layer->input_value(4).get_node_shared_ptr())
                      ->cast_vector<float>())});
-        ngraph::copy_runtime_info(new_fq, fq_layer);
+            copy_runtime_info(new_fq, fq_layer);
         return new_fq;
     }
     return last_node;
 }
+
+void RemoveSingleInputNodeFromFunction(std::shared_ptr<ov::Node> node) {
+    const ov::Shape input_node_shape = node->get_input_shape(0);
+    const ov::Shape output_node_shape = node->get_output_shape(0);
+
+    std::shared_ptr<ov::Node> node_parent = node->get_input_node_shared_ptr(0);
+    if (!std::equal(input_node_shape.begin(), input_node_shape.end(), output_node_shape.begin())) {
+        auto reshape_const_node =
+            std::make_shared<Constant>(ov::element::i64, ov::Shape{output_node_shape.size()}, output_node_shape);
+        node_parent = std::make_shared<Reshape>(node_parent, reshape_const_node, false);
+    }
+
+    ov::replace_output_update_name(node->output(0), node_parent->output(0));
+}
+
+ov::Shape SqueezeShape(const ov::Shape& shape) {
```

Comment:
We have functions operating on dims in util.hpp file already. I suggest to move it there and remove duplicates like e.g. IsOneDimLayer.